packages/core/src/lib/time-tracking/time-slot/commands/handlers/update-time-slot-minutes.handler.ts

```
@@ -7,26 +7,36 @@ import { TypeOrmTimeSlotMinuteRepository } from '../../repository/type-orm-time-
+	 * and returns the updated record with its `timeSlot` relation.
+	 *
+	 * @param command - Contains the ID of the time slot minute and updated input data.
+	 * @returns A Promise resolving to the updated `TimeSlotMinute` entity, or `null` if not found.
+	 */
 	public async execute(command: UpdateTimeSlotMinutesCommand): Promise<TimeSlotMinute> {
 		const { input, id } = command;
-		let timeMinute = await this.typeOrmTimeSlotMinuteRepository.findOneBy({
-			id
-		});
+
+		let timeMinute = await this.typeOrmTimeSlotMinuteRepository.findOneBy({ id });
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'timeMinute' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)